src/store/index.ts

```diff
@@ -391,16 +391,15 @@ export default createStore({
     batterySystemData: BatterySystemData,
     batterySystemMode: localStorage.getItem('el-demoapp-battery-system-mode') || 'real',
     batterySystem: JSON.parse(localStorage.getItem('el-demoapp-battery-system') || 'null') || BatterySystem as BatterySystem,
-    batterySystemPointC: 0xE7,
+    batterySystemPointC: 0xD2,
```

Comment:
@justdecodeme  The battery system logic change was not requested. Could you please revert it to its original state?